WordPress/src/main/java/org/wordpress/android/ui/sitecreation/theme/HomePagePickerFragment.kt

```diff
@@ -83,26 +81,23 @@ class HomePagePickerFragment : Fragment() {
             setupUi()
             setupViewModel()
             setupActionListeners()
-            previewModeSelectorPopup = PreviewModeSelectorPopup(
-                    requireActivity(),
-                    homePagePickerTitlebar.previewTypeSelectorButton
-            )
         }
     }
 
     private fun HomePagePickerFragmentBinding.setupUi() {
         homePagePickerTitlebar.title.isInvisible = !displayUtils.isPhoneLandscape()
-        modalLayoutPickerHeaderSection.modalLayoutPickerTitleRow?.header?.setText(R.string.hpp_title)
-        modalLayoutPickerHeaderSection.modalLayoutPickerSubtitleRow?.description?.setText(R.string.hpp_subtitle)
+        with(modalLayoutPickerHeaderSection) {
+            modalLayoutPickerTitleRow?.header?.setText(R.string.hpp_title)
+            modalLayoutPickerSubtitleRow?.root?.visibility = View.GONE
```

Comment:
It seemed simplest to me to just remove this from the layout. Alternatively, we could remove it in the xml, but since it is currently shared with the page picker, this would require forking the layouts. :man_shrugging:

Reply:
It also feels to me that your approach is the more convenient one 👍